### PR TITLE
Add orientation/axes cube

### PR DESCRIPTION
**Pull request by floryst**
Adds AnnotatedCubeActor and OrientationMarkerWidget, as well as ImageHelper.

### CODE REVIEW

File: Sources/Interaction/Widgets/OrientationMarkerWidget/example/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+axes.setZPlusFaceProperty({ text: '+Z', faceColor: '#ff00ff' });
+axes.setZMinusFaceProperty({ text: '-Z', faceColor: '#0000ff' });
+
+// create orientation widget
+const orientationWidget = vtkOrientationMarkerWidget.newInstance({
+  actor: axes,
+  interactor: renderWindow.getInteractor(),
+});
+orientationWidget.setEnabled(true);
+orientationWidget.setViewportSize(0.15);
```
Comment (jourdain) [inline Sources/Interaction/Widgets/OrientationMarkerWidget/example/index.js]: How do you specify it's location on the screen?

File: Sources/Interaction/Widgets/OrientationMarkerWidget/index.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
+import macro       from 'vtk.js/Sources/macro';
+import vtkRenderer from 'vtk.js/Sources/Rendering/Core/Renderer';
+
+// ----------------------------------------------------------------------------
+// vtkOrientationMarkerWidget
+// ----------------------------------------------------------------------------
+
+const CORNERS = {
```
Comment (jourdain) [inline Sources/Interaction/Widgets/OrientationMarkerWidget/index.js]: You may want to use the a Constants.js file for defining those constants.

**Comment by jourdain** [comment]
Looks good. 

But can you gather you API comments into an `api.md` file which give you more freedom for code snippet and more (if you feel like it).

Also those `api.md` are used for generating the documentation online.

**Comment by floryst** [comment]
Cool, I'll add an api.md.

**Comment by jourdain** [comment]
thank!

**Comment by floryst** [comment]
Added API docs, moved public constants to Constants.js, and minor code cleanups.

**Pull request merged**